python/Ganga/testlib/functions.py

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
I don't think that `functions.py` is a useful name for this file. `utils.py` or `file_utils.py` or similar describes the purpose much better.